Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:16.14.2-alpine3.15 As development
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+RUN npm run build
+
+CMD ["npx", "nest", "start", "--debug", "0.0.0.0:9229", "--watch"]
+
+FROM node:16.14.2-alpine3.15 as production
```

Comment:
This is fine for now but if we are to run Docker in production it will need to be extended quite a bit. We will also need to solve the DB credentials problem